connection.go

```diff
@@ -324,7 +325,8 @@ func (ch *Channel) newConnection(conn net.Conn, initialID uint32, outboundHP str
 		events:             events,
 		commonStatsTags:    ch.commonStatsTags,
 		healthCheckHistory: newHealthHistory(),
-		lastActivity:       *atomic.NewInt64(ch.timeNow().UnixNano()),
+		lastActivityRecv:   *atomic.NewInt64(ch.timeNow().UnixNano()),
```

Comment:
nit: can we call `ch.timeNow().UnixNano()` once and use the same value in both places -- that way they'll be the same value when a connection starts

Reply:
Done.